Formula/tomcat@8.0.rb

```diff
@@ -3,16 +3,15 @@ class TomcatAT80 < Formula
   homepage "https://tomcat.apache.org/"
 
   stable do
-    url "https://www.apache.org/dyn/closer.cgi?path=tomcat/tomcat-8/v8.0.42/bin/apache-tomcat-8.0.42.tar.gz"
-    sha256 "be876f955657791a480c167ea126663f56aa908da5c354bd4b3d60b3aa5c6e1e"
+    url "http://apache.mirrors.pair.com/tomcat/tomcat-8/v8.0.43/bin/apache-tomcat-8.0.43.tar.gz"
```

Comment:
Nevermind. You're right, there is some magic in the background that parses out proper URLs. Let me update PR.